compare_files.bash

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Function to compare two files
+compare_files() {
+    local file1=$1
+    local file2=$2
+
+    if ! cmp -s "$file1" "$file2"; then
+        echo "Files $file1 and $file2 are different."
+    fi
+}
+
+# Loop through numbers 1 to 99
+for i in $(seq 1 99); do
+    file_pg="/Users/qianqian/spiceai/crates/runtime/benches/snapshots/bench__file_tpcds_q$i.snap"
+    s3_pg="/Users/qianqian/spiceai/crates/runtime/benches/snapshots/bench__databricks_delta_tpcds_q$i.snap"
+
+    # Check if both files exist
+    if [[ -f "$file_pg" && -f "$s3_pg" ]]; then
```

Comment:
You should probably check they they have size > 0

Reply:
Ohhh I upload this by accident, will remove the file